src/component/tooltip/TooltipView.ts

```diff
@@ -945,16 +948,25 @@ class TooltipView extends ComponentView {
                     && lastItem.axisId === thisItem.axisId
                     && lastIndices.length === newIndices.length;
 
-                contentNotChanged && each(lastIndices, function (lastIdxItem, j) {
+                contentNotChanged && each(lastIndices, (lastIdxItem, j) => {
                     const newIdxItem = newIndices[j];
                     contentNotChanged = contentNotChanged
                         && lastIdxItem.seriesIndex === newIdxItem.seriesIndex
                         && lastIdxItem.dataIndex === newIdxItem.dataIndex;
                 });
+
+                // check is cbParams data value changed
+                lastCbParamsList && zrUtil.each(lastItem.seriesDataIndices, (idxItem) => {
+                    const cbParams = cbParamsList[idxItem.dataIndexInside];
```

Comment:
It should be `idxItem.seriesIndex` here. It includes data of all series on one `dataIndex`.  Also, we need to check if cbParams is null because the length of the last indices may not match the current.

Reply:
fixed